### DISCUSSION TITLE
Welcome to Planetary Computer Discussions!

### DISCUSSION THREAD

**Discussion by lossyrob** [microsoft/PlanetaryComputer#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with members of the Planetary Computer community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do or would like to do with the Microsoft Planetary Computer!

**Comment by Ethan-Chen421** [comment]
I am using the API endpoint of MPC and don't know how to obtain the tile data of the image; For example, the image tile data of S2B-MSIL2A_20230622T023529-R089-T52TCT20230622T143158

**Reply by TomAugspurger** [reply]
https://planetarycomputer.microsoft.com/docs/quickstarts/reading-stac/ and https://planetarycomputer.microsoft.com/docs/concepts/sas/ might be worth reading through, if you haven't already.
